canonical-web-and-design/ubuntu.com
# Builds telco/openRAN page

## Done

- Builds telco/openRAN page

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/telco/openran
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare against [copy-doc](https://docs.google.com/document/d/181cqeYzSZ_RinjPTMMNDyAEMwKVvdeoc30iQDrwl4aQ/edit#)


## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4192
